azurerm/data_source_healthcare_service_test.go

```diff
@@ -0,0 +1,46 @@
+package azurerm
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform/helper/acctest"
+)
+
+func TestAccAzureRMDataSourceHealthcareService(t *testing.T) {
+	dataSourceName := "data.azurerm_healthcare_service.test"
+	ri := acctest.RandIntRange(10000, 99999)
```

Comment:
Could we use the same function as the rest of the tests here:
```suggestion
	ri := tf.AccRandTimeInt()
```

Reply:
I resrticted thsi random int because this value is used in fhir service name. But name can not be more 24 chars. So I have to use shorter number.

Reply:
this is still a concern